specs/subresourceintegrity/spec.markdown

```diff
@@ -285,6 +285,9 @@ collision-resistant.  For example, `getPrioritizedHashFunction('sha256',
 'sha512')` would return `'sha512'` and `getPrioritizedHashFunction('sha256',
 'sha256')` would return the empty string.
 
+The `getPrioritizedHashFunction` implementation is vendor specific, and the interface 
```

Comment:
The verb tense here doesn't feel quite right. The previous paragraph says "if a user agent implemented getPrioritizedHashFunction() then it would ..." and in this change, it feels like we assume that it exists.

I suggest rewording to something like "Note that getPrioritizedHashFunction() is never exposed to applications given that both its implementation and its existence are vendor-specific."